### ISSUE TITLE
mbstring.internal_encoding warning

### ISSUE THREAD

**Issue by phaidon** [zikula/core#2197]
The use of mbstring.internal_encoding cause a  warning on my page:

 ```
Deprecated: ini_set(): 
Use of mbstring.internal_encoding is deprecated in Zikula/src/lib/legacy/Zikula/Core.php on line 30
```
mac os x - php 5.6.2

**Comment by craigh** [comment]
This won't be changed until at least Core 2.0.0

**Comment by phaidon** [comment]
Sounds reasonable.

**Comment by shefik** [comment]
mbstring.internal_encoding string

Warning

This feature has been DEPRECATED as of PHP 5.6.0. Relying on this feature is highly discouraged.
Defines the default internal character encoding.

PHP 5.6 and later users should leave this empty and set default_charset instead.

http://php.net/manual/en/mbstring.configuration.php#ini.mbstring.internal-encoding

**Comment by craigh** [comment]
closed by #2199
